setup.cfg

```diff
@@ -13,8 +13,8 @@ classifiers =
     Intended Audience :: Developers
     License :: OSI Approved :: Apache Software License
     Operating System :: OS Independent
-    Programming Language :: Python :: 2.7
     Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.8
```

Comment:
@morenol remove this. Tests are not running on 3.8.

Reply:
You can just drop 2.7.
Add py3.8 with allowed failure in travis without py2.8 classifiers in this PR.
create new PR for py3.8 failures fix.

Reply:
I think that the failure of the test was not related to this change https://travis-ci.org/github/edx/edx-lint/jobs/683180550, I triggered them again and they are ok

Reply:
Please review this again